guides/source/configuring.md

```diff
@@ -89,6 +90,7 @@ Below are the default values associated with each target version. In cases of co
 - [`config.active_storage.variant_processor`](#config-active-storage-variant-processor): `:vips`
 - [`config.action_controller.wrap_parameters_by_default`](#config-action-controller-wrap-parameters-by-default): `true`
 - [`config.action_dispatch.default_headers`](#config-action-dispatch-default-headers): `{ "X-Frame-Options" => "SAMEORIGIN", "X-XSS-Protection" => "0", "X-Content-Type-Options" => "nosniff", "X-Download-Options" => "noopen", "X-Permitted-Cross-Domain-Policies" => "none", "Referrer-Policy" => "strict-origin-when-cross-origin" }`
+- [`config.active_record.encryption.hash_digest_class`](#config-active-encryption-hash-digest-class): `OpenSSL::Digest::SHA1`
```

Comment:
I think instead of putting the original value in older defaults, you should create a markdown table showing the difference:

https://github.com/rails/rails/blob/255accb0a584962394330810b07300b7fa7a9aee/guides/source/configuring.md?plain=1#L175-L178

Reply:
The table in the config description looks good, but I think this line still needs to be removed